kibble/__main__.py

```diff
@@ -15,9 +15,83 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import click
+
+from kibble.cli import setup_command
+from kibble.version import version as kv
```

Comment:
```suggestion
from kibble.version import version as kibble_version
```
How about more meaningful name?